### ISSUE TITLE
Morning session suggestion

### ISSUE THREAD

**Issue by kessie2862** [M-AminAlizadeh/Portfolio#8]
A website with a good accessibility rating must be able to navigate through the page, focus its elements, and select and navigate through its links using only the keyboard. When I try to navigate the mobile version of your web page using the `tab` key, the menu icon does not receive focus and this is because you are not using focusable elements such as an `a `tag. I suggest you kindly fix this so as to improve the page accessibility.

![accessibility-img](https://user-images.githubusercontent.com/97234029/230309707-5519ba5d-f1b5-429e-b829-637d34c59bc5.png)
